### ISSUE TITLE
Landscape Mode

### ISSUE THREAD

**Issue by xmha97** [meypod/al-azan#18]
Hi,
Please add support landscape mode.
Thanks.

**Comment by meypod** [comment]
Hi
I don't know if it's going to be easy
But can you tell me what do you have in mind ?
Like how the app you think should look like in landscape mode ?
Also can you tell me what's your device ?

**Comment by xmha97** [comment]
Usually, people work with phones in portrait mode and with tablets in landscape mode.
I also want to install this app on a tablet and use it in landscape mode (like other apps)
My device is "Samsung Galaxy Tab S6"

**Comment by meypod** [comment]
I'll give it a go when I have time

**Comment by meypod** [comment]
I've tried making it break into two columns using flex wrap and were forced to disable scroll for it to work; while it would work under "Normal" conditions, it would break for people who use extra large everything settings of android (I mean Large Font Size + Display Size). and I know these people in real life and I know It would be impractical for them. so I was stuck between limited choices, so I decided to go with "make it work for everyone" and simply only "unlocked" screen orientation.
so while it would look ugly, it would still work for all people.
I'm still open to ideas that don't drastically change the UI and make this happen

I'll release the fix soon
